paddle/fluid/operators/conv_transpose_op.cc

```diff
@@ -168,8 +168,14 @@ void Conv2DTransposeOpMaker::Make() {
   AddAttr<bool>("use_mkldnn",
                 "(bool, default false) Only used in mkldnn kernel")
       .SetDefault(false);
-  AddAttr<bool>("fuse_relu", "(bool, default false) Only used in mkldnn kernel")
-      .SetDefault(false);
+  AddAttr<std::string>("fuse_activation",
```

Comment:
Please do not remove the existing attributes. 
![image](https://user-images.githubusercontent.com/6836917/62848483-7c0e1400-bd0e-11e9-9900-29bbecdae27a.png)https://www.paddlepaddle.org.cn/documentation/docs/en/1.5/advanced_usage/development/new_op/op_notes_en.html

Reply:
Is it sufficient just to leave them as there were but without using them?

Reply:
Yes, it is.